brettdottech/info-orbs
# WiFi Manager

Trello Card: https://trello.com/c/62TXyOWv/43-build-a-basic-companion-webserveror-other-better-app-solution

## Changes
This does a few things:
On startup that checks a SPIFFS file system for stored WiFi credentials. 
If none exist, an Access Point is started for you to connect to under the name "InfoOrbs-WiFi-Manager"
If credentials are stored, it connects the orbs to your WiFi network to run a simple web server to configure settings for the device.

The settings in the web server step currently do nothing, but I wanted to get the base of the work in first before this PR got to be too massive. I will create a follow up PR with that functionality in it

## Testing
* Connect to the manager wifi network and go to the IP shown on your orbs
* Enter in your information in the UI provided
  * The IP is the device IP to bind to after setup
  * The Gateway is the IP of your router - in most cases this will be 192.168.1.1 or 192.168.0.1
* Once configured and the orbs have reset, check that they have continued onto your first widget
* Go to the device IP you configured in the first step
* You should then see the web server settings
